AEPServices/Mocks/PublicTestUtils/MockNetworkService.swift

```diff
@@ -108,58 +118,30 @@ public class MockNetworkService: Networking {
-            return
-        }
-        helper.setExpectation(for: networkRequest, expectedCount: expectedCount, file: file, line: line)
-    }
-
-    /// Asserts that the correct number of network requests were seen for all previously set network request expectations.
-    ///
-    /// - Parameters:
-    ///   - ignoreUnexpectedRequests: A Boolean value indicating whether unexpected requests should be ignored. Defaults to `true`.
-    ///   - timeout: The time interval to wait for network requests before timing out. Defaults to ``TestConstants/Defaults/WAIT_NETWORK_REQUEST_TIMEOUT``.
-    ///   - file: The file from which the method is called, used for localized assertion failures.
-    ///   - line: The line from which the method is called, used for localized assertion failures.
-    /// - SeeAlso:
-    ///     - ``setExpectationForNetworkRequest(url:httpMethod:expectedCount:file:line:)``
-    public func assertAllNetworkRequestExpectations(ignoreUnexpectedRequests: Bool = true, timeout: TimeInterval = TestConstants.Defaults.WAIT_NETWORK_REQUEST_TIMEOUT, file: StaticString = #file, line: UInt = #line) {
-        helper.assertAllNetworkRequestExpectations(ignoreUnexpectedRequests: ignoreUnexpectedRequests, timeout: timeout, file: file, line: line)
-    }
-
     /// Immediately returns all sent network requests (if any) **without awaiting**.
     ///
     /// Note: To await network responses for a given request, make sure to set an expectation
     /// using ``setExpectationForNetworkRequest(url:httpMethod:expectedCount:file:line:)``
     /// then await the expectation using ``assertAllNetworkRequestExpectations(ignoreUnexpectedRequests:timeout:file:line:)``.
     ///
-    /// - Returns: An array of `NetworkRequest` objects representing all sent network requests.
+    /// - Returns: An array of ``NetworkRequest`` objects representing all sent network requests.
     public func getNetworkRequests() -> [NetworkRequest] {
         return helper.orderedNetworkRequests
     }
 
-    /// Returns the network request(s) sent through the Core NetworkService, or empty if none was found.
+    /// Returns all network requests that match the provided network request using ``TestableNetworkRequest/isEqual(_:)``.
     ///
-    /// Use this method after calling `setExpectationForNetworkRequest(url:httpMethod:expectedCount:file:line:)` to wait for expected requests.
+    /// This method waits for the specified network request to be fulfilled within the provided timeout interval,
+    /// ensuring that any expected requests are received. If no expectation exists for the provided request,
+    /// the method will perform a regular wait for the specified timeout duration.
     ///
     /// - Parameters:
-    ///   - url: The URL `String` of the `NetworkRequest` to get.
-    ///   - httpMethod: The HTTP method of the `NetworkRequest` to get.
+    ///   - url: The URL ``String`` of the ``NetworkRequest`` to get.
```

Comment:
code formatting is being applied twice for String and Network request and similar to other function doc updates that have been done.

Reply:
Got it. Nothing to be done.
https://stackoverflow.com/questions/27715933/how-to-use-swift-documentation-comments